src/App.js

```diff
@@ -96,17 +111,20 @@ class App extends Component {
 
   render() {
     const isLoggedIn = !!this.state.user;
-    if (isLoggedIn) {
-      return (
-        <div>
-          <Header />
-          <img src={logo} alt="logo" />
-          <Footer />
-        </div>
-      );
-    } else {
-      return <div>Loading...</div>;
-    }
+    const i18n = this.state.i18n;
+    return (
+      <IntlProvider locale={i18n.locale} messages={i18n.messages}>
+        {isLoggedIn ? (
+          <div>
```

Comment:
maybe can change for `<>`, what do you think?

Reply:
I am fine, I did not want to change your layout, but if `<>` is fine, great.